connect-inject/health_check_resource.go

```diff
@@ -221,8 +220,7 @@ func (h *HealthCheckResource) getServiceCheck(client *api.Client, healthCheckID
 	filter := fmt.Sprintf("CheckID == `%s`", healthCheckID)
 	checks, err := client.Agent().ChecksWithFilter(filter)
 	if err != nil {
-		h.Log.Error("unable to get agent health check", "checkID", healthCheckID, "filter", filter, "err", err)
```

Comment:
Is there a reason to log/exclude this `filter` field? Noticed it's getting removed from the error message and want to make sure its not useful.

Reply:
good question on this, I think it's probably okay to not log the filter because it's just a concatenation of "CheckID ==" and the healthCheckID which we're returning in the error. The front end of this string should never change.

Reply:
Yeah exactly. Also the error will actually include the url path which does include the filter.

My overall thinking was that the users don't need to know the filter because they won't understand what it is.